.github/workflows/ci.yml

```diff
@@ -89,7 +55,6 @@ jobs:
       run: cabal haddock
 
   build-freebsd:
-    needs: build
```

Comment:
It was just a guard to avoid launching the FreeBSD build when other builds fail, it is optional.  Not sure why @Bodigrim removed it, but it is optional. Removing it likely makes CI complete faster when all goes well, keeping it may mean that the FreeBSD build runs even on broken PRs.